src/main/java/edu/cornell/kfs/gl/batch/service/impl/CuPostExpenditureTransaction.java

```diff
@@ -0,0 +1,53 @@
+package edu.cornell.kfs.gl.batch.service.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kuali.kfs.coa.businessobject.Account;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryExclusionAccount;
+import org.kuali.kfs.coa.businessobject.ObjectCode;
+import org.kuali.kfs.gl.batch.service.impl.PostExpenditureTransaction;
+import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+public class CuPostExpenditureTransaction extends PostExpenditureTransaction {
+	
+	private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(CuPostExpenditureTransaction.class);
+	public static final String ACTIVE_INDICATOR_FIELD = "ACTV_IND";
+	
+	public CuPostExpenditureTransaction() {
+		super();
+	}
+	
+	/**
+     * Determines if there's an exclusion by account record for the given account and object code
+     * @param account the account to check
+     * @param objectCode the object code to check
+     * @return true if the given account and object code have an exclusion by account record, false otherwise
+     */
+    public boolean hasExclusionByAccount(Account account, ObjectCode objectCode) {
```

Comment:
If you're overriding a superclass method, I would suggest adding the "@Override" annotation for clarity.